Lib/test/test__interpreters.py

```diff
@@ -0,0 +1,506 @@
+import contextlib
+import os
+import os.path
+import shutil
+import tempfile
+from textwrap import dedent, indent
+import threading
+import unittest
+
+from test import support
+from test.support import script_helper
+
+interpreters = support.import_module('_interpreters')
+
+
+SCRIPT_THREADED_INTERP = """\
+from textwrap import dedent
+import threading
+import _interpreters
+def f():
+    _interpreters.run_string(id, dedent('''
```

Comment:
Good catch.  That's a typo that I didn't notice because the resulting exception in the thread does not cause the process to fail.

FYI, adding the missing "id = _interpreters.create()" line causes the test to fail (correctly); the subinterpreter didn't get cleaned up before we reached Py_Finalize(), resulting in Py_FatalError.  I'll sort that out separately.